packages/lib/services/interop/InteropService_Importer_Md.ts

```diff
@@ -110,11 +110,18 @@ export default class InteropService_Importer_Md extends InteropService_Importer_
 		const htmlLinks = htmlUtils.extractFileUrls(md);
 		const fileLinks = unique(markdownLinks.concat(htmlLinks));
 		for (const encodedLink of fileLinks) {
-			const link = decodeURI(encodedLink);
+			let link = '';
+			try {
+				link = decodeURI(encodedLink);
+			} catch (error) {
+				// If the URI cannot be decoded, leave it as it is.
+				continue;
+			}
 
 			if (isDataUrl(link)) {
 				// Just leave it as it is. We could potentially import
 				// it as a resource but for now that's good enough.
+				continue;
```

Comment:
Any reason why a `continue` also needs to be added here? It seems unrelated to the original bug since this is to deal with data URLs?

Reply:
I added it at Line 124 because I want to make the style consistent. The `continue` at [Line 118](https://github.com/laurent22/joplin/pull/11576/files#diff-3d2b9c741c147d26db4122ce4fd7808caaf1df0a6ae872ce69ac7c7e0e5dec9eR118) is for excluding any further processing to the malformed URLs. I think this also applies to the data URLs so I add a continue here.
The extra `continue` at Line 124 doesn't affect the existing logic because all the following processing is in the `else {...}` block. In fact, we can even remove the `else` statement here because the following statements in the `for` loop is skipped.
If you feel it's inappropriate to add `continue` at Line 124, I can also remove it.